tests/test_basic_ops_dict.py

```diff
@@ -203,6 +200,30 @@ def test_dict_get_with_default(
         OmegaConf.set_struct(c, struct)
         assert c.get(key, default_val) == default_val
 
+    @pytest.mark.parametrize(
+        ("d", "select", "key", "expected"),
+        [
+            ({"hello": "world"}, "", "hello", "world"),
+            ({"hello": None}, "", "hello", None),
+            ({"hello": {"a": None}}, "hello", "a", None),
```

Comment:
hello is not very informative.
```suggestion
            ({"key": "value"}, "", "key", "value"),
            ({"key": None}, "", "key", None),
            ({"key": {"a": None}}, "key", "a", None),
```

etc.

Reply:
Done in eecac80